install_template/templates/products/edb-postgres-advanced-server/_centos-rhel-installcommand.njk

```diff
@@ -1 +1 @@
-Installing the server package creates an operating system user named `enterprisedb`. The user is assigned a user ID (UID) and a group ID (GID). The user has no default password. Use the `passwd` command to assign a password for the user. The default shell for the user is `bash` and the user's home directory is `/var/lib/edb/as{{ product.version }}`.
\ No newline at end of file
+Installing the server package creates an operating system user named enterprisedb. The user is assigned a user ID (UID) and a group ID (GID). The user has no default password. Use the `passwd` command to assign a password for the user. The default shell for the user is `bash` and the user's home directory is `/var/lib/edb/as{{ product.version }}`.
```

Comment:
Per our recent discussion in our team channel we aren't using a font treatment for user names as is reflected in this edit. The font treatment is showing up in the generated topics, however. Something seems amiss.